scripts/common.py

```diff
@@ -98,8 +100,10 @@ class Config:
     CC_DB_JSON = "compile_commands.json"
 
     # Look up rust toolchain from repo root
-    with open(os.path.join(ROOT_DIR, "rust-toolchain")) as fh:
-        CUSTOM_RUST_NAME = fh.readline().strip()
+    rust_toolchain_str = Path(ROOT_DIR).joinpath("rust-toolchain.toml").read_text()
+    rust = toml.loads(rust_toolchain_str)
+    CUSTOM_RUST_NAME = rust["toolchain"]["channel"]
```

Comment:
We can probably extract this into its own `.py` somewhere to use as a utility in the shell scripts.

Reply:
Fixed in https://github.com/immunant/c2rust/pull/520/commits/f8793f0ad885c207bb6be24d3f7bd50bffb7beda.

I extracted this into `query_toml.py` (a bit more generic) and then used it here.